### ISSUE TITLE
Add option to set a basepath for every path

### ISSUE THREAD

**Issue by jormaechea** [danielgtaylor/apisprout#26]
**Summary**
If servers url contain a base path (for example: `https://example.com/api`), when you mock it you loose that base path.

**Steps to reproduce**
Mock the following schema:
```
openapi: 3.0.0
info:
  description: Example API
  title: Example API
  version: 1.0.0

servers:
  - url: 'https://example.com/api'
    description: The Example API server

paths:
  /foo:
    get:
      operationId: getFoo
      responses:
        '200':
          content:
            application/json:
              schema:
                type: object
                properties:
                  bar:
                    type: string

```

**Actual result**
The API endpoint is mocked as `http://localhost/foo`

**Expected result**
The API endpoint should be mocked as `http://localhost/api/foo`

**More info**
Suggested solutions
a) Add an optional argument to include `server.url` basepath in every endpoint (eg: `--include-base-path` for cli)
b) Add an optional argument to define a basepath for every endpoint (eg: `--basepath=/api` for cli)

**Comment by danielgtaylor** [comment]
Note: there is a PR which can solve this but needs testing in #14

**Comment by danielgtaylor** [comment]
Closing as this was fixed in 06d16640d6cd8608b83e8d15dcd05be76117f968
